<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_Dialog</class>
 <widget class="QDialog" name="Image_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据图片</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>384</width>
       <height>288</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
